pkg/stub/handler.go

```diff
@@ -152,6 +152,7 @@ func (h *Handler) processImageStreamWatchEvent(is *imagev1.ImageStream) error {
 	defer h.mutex.Unlock()
 
 	srcfg, filePath, ok := h.prepWatchEvent("imagestream", is.Name, is.Annotations)
+	logrus.Debugf(" prep watch event imgstr %s ok %v", is.Name, ok)
```

Comment:
if you're cleaning things up you've got a variety of prefix spaces in these messages throughout